projects/gpu_operator/toolbox/gpu_operator_run_gpu_burn/files/gpu-burn/gpu_burn-drv.cpp

```
@@ -1,621 +1,944 @@
+    ss_model << f_model.rdbuf();
+    printf("%s\n", ss_model.str().c_str());
+#else
+    system("nvidia-smi -L");
+#endif
+
+    // Initting A and B with random data
+    T *A = (T *)malloc(sizeof(T) * SIZE * SIZE);
+    T *B = (T *)malloc(sizeof(T) * SIZE * SIZE);
+    srand(10);
+    for (size_t i = 0; i < SIZE * SIZE; ++i) {
+        A[i] = (T)((double)(rand() % 1000000) / 100000.0);
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)